frontend/src/app/signUp/IdentityVerification/PersonalDetailsForm.tsx

```diff
@@ -95,6 +95,12 @@ const PersonalDetailsForm = ({
     );
     showNotification(alert);
     setSaving(false);
+    let elementsWithErrors = Array.from(
+      document.querySelectorAll("[aria-invalid=true]")
+    );
+    (elementsWithErrors.find(
+      (element) => element.getAttribute("aria-hidden") !== "true"
+    ) as HTMLElement | null)?.focus();
```

Comment:
It may be worth moving this block of code into one of our of common `utils` files, because I imagine this is nearly universally true on all our forms - we want to bring focus on the first visible `aria-invalid` object. That would enable you to reuse it between these two forms, but also allow future work like #4094 to reuse it as well.